eth/ethconfig/config.go

```diff
@@ -50,9 +50,9 @@ var FullNodeGPO = gasprice.Config{
 // Defaults contains default settings for use on the Ethereum main net.
```

Comment:
Should we change the comment, since this is not about the Ethereum main net anymore?

Reply:
Fixed that in another commit